### ISSUE TITLE
Duplicated entries in process log

### ISSUE THREAD

**Issue by mckayERP** [adempiere/adempiere#2453]
When a process is run, the result message is often duplicated.

![image](https://user-images.githubusercontent.com/5507616/55243471-b2db0600-5215-11e9-9095-345c577bc30b.png)

The problem is caused by a call to ```ProcessInfoUtil.setLogFromDB(ProcessInfo pi)``` which adds the log entries even if the log entries already exists.

For speed and to avoid the duplication, I'd propose testing the log for entries and not updating the log if any exist.

Basically add the following check to setLogsFromDB(pi)

		// #2453 Check if the log already exists - don't overwrite or duplicate
		if (pi.getLogList() != null && pi.getLogList().size() > 0)
			return;
		
The result is

![image](https://user-images.githubusercontent.com/5507616/55244360-6f819700-5217-11e9-948f-00a9cc1d160c.png)
